controllers/ajax.py

```diff
@@ -140,7 +147,6 @@ def hsblog():
             sid=sid, answer=act, div_id=div_id, timestamp=ts, course_name=course)
 
     response.headers['content-type'] = 'application/json'
-    res = {'log':True}
```

Comment:
It may just be that the test itself needs to be updated to reflect your addition of the timestamp to the default??

Reply:
Yes, that's all. I'll work on this when I can, probably next week.

Reply:
Why did you need to add timestamp to the return value??

Reply:
Thanks for taking a look!

Client-side [code](https://github.com/RunestoneInteractive/RunestoneComponents/blob/master/runestone/common/js/runestonebase.js#L115) compares the server's date with the client's date to determine which is newer. If the client's clock isn't set correctly, this comparison will return an incorrect value. So, I'm having the server provide the time, then using this in client code to record the time, rather than the client's clock.